src/components/LowerRightControls.tsx

```diff
@@ -88,7 +88,7 @@ export function LowerRightControls({
         <Link
           to={
             location.pathname.includes(paths.FILE)
-              ? filePath + paths.SETTINGS
+              ? filePath + paths.SETTINGS + '?tab=project'
```

Comment:
can we keep these magic strings in constants so the code which checks for them and the code which sets them are always using the same value? We have so many typos in our strings haha.

Reply:
yea was just copying what we do elsewhere